geonode/security/utils.py

```diff
@@ -324,21 +274,348 @@ def get_user_visible_groups(user, include_public_invite: bool = False):
     return metadata_author_groups
 
 
-def get_obj_group_managers(owner):
-    """
-    Returns the managers of all the groups belonging to the "owner"
-    """
-    obj_group_managers = []
-    if get_user_groups(owner):
-        for _user_group in get_user_groups(owner):
-            if not skip_registered_members_common_group(Group.objects.get(name=_user_group)):
-                try:
-                    _group_profile = GroupProfile.objects.get(slug=_user_group)
-                    managers = _group_profile.get_managers()
-                    if managers:
-                        for manager in managers:
-                            if manager not in obj_group_managers and not manager.is_superuser:
-                                obj_group_managers.append(manager)
-                except GroupProfile.DoesNotExist as e:
-                    logger.exception(e)
-    return obj_group_managers
+class AdvancedSecurityWorkflowManager:
+
+    @staticmethod
```

Comment:
All the static methods that start with `is_` look like more properties than methods

Reply:
I would prefer to keep them static so I don't need to create an instance of the `AdvancedSecurityWorkflowManager` everytime